docs/integration/telegraf.txt

```diff
@@ -0,0 +1,66 @@
+===============================
+Using Telegraf with CrateDB
+===============================
+
+`Telegraf <https://github.com/influxdata/telegraf>`_ is a plugin-driven server agent for collecting & reporting metrics.
```

Comment:
okay so for this intro section I think we need a bit more info

can you add a para saying why you might want to use CrateDB with Telegraf (use case, etc) and what this particular doc is going to show you how to do and for what reason

Reply:
Is this enough? or feel free to add something.

Reply:
this is fin